### DISCUSSION TITLE
monitor xx is not in quorum list?

### DISCUSSION THREAD

**Discussion by zhucan** [rook/rook#9881] [Q&A]
![image](https://user-images.githubusercontent.com/13997135/157616296-564bdac9-b2bc-4536-9f48-37da30c46c54.png)

**Comment by zhucan** [comment]
@travisn

**Comment by travisn** [comment]
@zhucan You may have a network issue. See [this topic](https://rook.github.io/docs/rook/v1.8/ceph-common-issues.html#monitors-are-the-only-pods-running) to troubleshoot. In the future, if you could also attach text logs it would be helpful instead of screenshots.

**Comment by zhucan** [comment]
@travisn Thanks, I wiill take a look.
